dev/init-clindb.sql

```diff
@@ -1 +1,3 @@
 -- MariaDB initialization script; run on first start only
+CREATE DATABASE IF NOT EXISTS radar;
+GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, INDEX, ALTER ON radar TO 'db-user'@'localhost';
```

Comment:
I wonder if it makes sense here to have two users with DB access -- a dbadmin account which can insert, update, delete, create, drop, and alter, and a less privileged dbuser account used by Leaf to select and index only?

Does the account Leaf uses for queries need permission to modify tables, or just select/join on them?

Reply:
>I wonder if it makes sense here to have two users with DB access -- a dbadmin account which can insert, update, delete, create, drop, and alter, and a less privileged dbuser account used by Leaf to select and index only?

That seems reasonable- the MariaDB docker container makes it easy to configure an admin (`root`) and another user. Since the admin is already configured, I'll tighten the privileges on the `db-user` account. 

Reducing the privileges could also help prevent  accidentally changing the study data.

>Does the account Leaf uses for queries need permission to modify tables, or just select/join on them?

I tested it out and as far as I can tell, only `select` and `CREATE TEMPORARY TABLES`

Addressed in 36cf5962edfab87fc654a88d0361e3ef03c3fda5